Utilities/StorageFactory/interface/StorageFactory.h

```diff
@@ -45,22 +53,27 @@ namespace edm::storage {
     std::string tempPath(void) const;
     double tempMinFree(void) const;
 
+    void setStorageProxyMakers(std::vector<std::unique_ptr<StorageProxyMaker>> makers);
+
     void stagein(const std::string &url) const;
-    std::unique_ptr<Storage> open(const std::string &url, int mode = IOFlags::OpenRead) const;
+    std::unique_ptr<Storage> open(const std::string &url, const int mode = IOFlags::OpenRead) const;
```

Comment:
Why `const int`? The parameter can't meaningfully be modified anyway.

Reply:
I'm being a bit over-cautious here. I wanted to make sure nothing in the body of `StorageFactory::open()` modifies the `mode` (it is being passed to function(s), so to make sure they don't take it by non-`const` reference)